### DISCUSSION TITLE
Purchasing a license

### DISCUSSION THREAD

**Discussion by korpthidrandir** [CoreForms/docs#2] [General]
Hi,

How would I go about purchasing a license to CoreForms? I've emailed coreforms@rubicon.eu twice (2023-08-24 and 2023-10-04), and called the number on the website (2023-10-11) which got me in contact with the office of Rubicon in general, but didn't help me to contact anyone related to this project.

Sorry if this is the wrong forum for this - I'm very excited about this technology, and would like to be able to use it.

Best regards,
Korp Thidrandir

**Comment by MichaelKetting** [comment]
Hi @korpthidrandir, thank's for reaching out, I just sent you a reply via mail.

Best Regards, Michael
